enterprise/server/remote_execution/snaploader/snaploader_test.go

```diff
@@ -283,6 +283,100 @@ func TestSnapshotVersioning(t *testing.T) {
 	}
 }
 
+func TestSnapshotVersioning(t *testing.T) {
+	for _, remoteEnabled := range []bool{true, false} {
+		flags.Set(t, "executor.enable_remote_snapshot_sharing", remoteEnabled)
+
+		ctx := context.Background()
+		env := setupEnv(t)
+		loader, err := snaploader.New(env)
+		require.NoError(t, err)
+		workDir := testfs.MakeTempDir(t)
+
+		// Task and configuration hash stay consistent, so that the only thing
+		// changing in the snapshot key is the version
+		task := &repb.ExecutionTask{}
```

Comment:
if `remoteEnabled` is true should we also make this a CI runner task so that we exercise the remote sharing codepath? or set `--debug_force_remote_snapshots`

Reply:
The ci_runner check (and `debug_force_remote_snapshots`) are applied at the firecracker level, in order to generate the `remoteEnabled` boolean to pass to snaploader code. The snaploader tests shoudn't need to worry about that because we can just directly pass the boolean